### PR TITLE
Integrate clothing parser into current workflow

### PR DESCRIPTION
**Pull request by danielheyman**
Added clothing parser to the current forum workflow. Also fixed new forum creation which was crashing due to the label array requirement.
 
screen shot 2017-08-21 at 11 40 30 pm

screen shot 2017-08-21 at 11 40 06 pm

### CODE REVIEW

File: routes/forums.js [hunk 1]
```diff
@@ -48,24 +48,23 @@ router.post('/', (req, res) => {
...
+    } else {
+        labels = labels.split(",");
     }
 
-    forumsData.addForum(title, content, label, clothing, userId)
+    forumsData.addForum(title, content, labels, userId)
         .then((newForum) => {
             return res.redirect(`/forums/${newForum._id}`);
         }).catch((err) => {
+        console.log("hi", err);
```
Comment (tsangjustin) [inline routes/forums.js:67]: Remove this console log

File: routes/forums.js [hunk 2]
```diff
@@ -77,6 +76,9 @@ router.get('/:forum_id', (req, res) => {
         .then((forumData) => {
             info.forum = forumData;
             console.log(info);
+            info.forum.contentHTML = forumData.content
+                .replace(/#([^\[]+)\[([^\]]+)\]/g, (match, name, url) => `<a target='_blank' alt='${name}' href='${url}'>${name}</a>`)
+                .replace(/@([\w-]+)/g, (match, username) => `<a target='_blank' alt='${username}' href='#'>${username}</a>`);
```
Comment (tsangjustin) [inline routes/forums.js:81]: Where would username link to?

**Comment by tsangjustin** [comment]
Been able to play around with websites, really cool!

File: routes/forums.js (hunk 2 above)
Comment (danielheyman) [reply routes/forums.js:81]: @tsangjustin for now nowhere. But just a cool reach features to have. Nice to see the usernames highlighted

**Pull request merged**